content/en/usage/setup.md

```diff
@@ -1,7 +1,99 @@
+
+For iOS, additional setup steps are only necessary under Windows. Here, you have to install the Apple Device Driver and the Apple Application Support. You can get those by installing iTunes.
+
+## Device preparation
+
+Our libraries work with physical phones and emulators on Android and physical phones on iOS. We test with the following target (but other versions may also work):
+
+| Platform | Type | Tested versions |
+| --- | --- | --- |
+| Android | `device` (Moto G7 Power) | 13 (API level 33) |
+| Android | `emulator` | 11 (API level 30), 13 (API level 33) |
+| iOS | `device` (iPhone X, iPhone 6S) | 15.6.1, 16.0 |
+
+Depending on what kind of device you want to use for the analysis, the preparation steps described below are necessary. Installing and setting up all other dependencies on the device is done automatically by our tools.
+
+### Physical Android phones
+
+On physical Android phones, USB debugging must be enabled. This can be done via *Settings* -> *System* -> *Developer options* (to activate, tap *Build number* seven times under *Settings* -> *About phone* -> *Android version*) -> *USB debugging*.
+
+To be able to do traffic analysis in a meaningful way, the device needs to be rooted. For this, we recommend [Magisk](https://topjohnwu.github.io/Magisk/). To use that, you also have to unlock the bootloader. The instructions for that vary from device to device. Usually, all data on the device is lost in the process.  
+After rooting, root debugging should be enabled if available: *Settings* -> *System* -> *Developer options* -> *Rooted debugging*.
+
+When connecting via USB, you have to confirm that you trust the computer.
+
+### Android emulator
+
+No special setup is required to use an Android emulator. You just have to create an emulator. This can be done through the [Android Studio](https://developer.android.com/studio) Device Manager or via the tweasel CLI, we recommended you configure larger storage space:
+
+```sh
+tweasel android-emulator:create "<emulator name>" --partition-size 16384
+```
+
+The first run may take a moment as the Android SDK is automatically set up in the background. The command then interactively asks what kind of emulator should be created. For maximum compatibility with apps [we currently recommend](https://github.com/tweaselORG/appstraction/issues/54): Android 11 (API level 30), Android with Google APIs, x86_64.
+
+If you want to set up the emulator further (for example to place honey data), you can start it as follows and then create a snapshot:
+
+```sh
+tweasel android-emulator:start "<emulator name>"
+tweasel android-emulator:snapshot:create "<snapshot name>" 
+```
+
+### Physical iPhones
+
+iOS devices also require a jailbreak. Our tools are tested on iOS 15 and 16 with the [palera1n jailbreak](https://github.com/palera1n/palera1n). Follow [this guide](https://ios.cfw.guide/installing-palera1n/). Important: The jailbreak must be installed in rootful mode.[^ios-14]
```

Comment:
Maybe we should also recommend `palen1x` considering how many problems we encountered…